wanaku-router/wanaku-router-backend/src/main/java/ai/wanaku/backend/resolvers/WanakuPromptsResolver.java

```diff
@@ -0,0 +1,34 @@
+package ai.wanaku.backend.resolvers;
+
+import ai.wanaku.api.exceptions.PromptNotFoundException;
+import ai.wanaku.api.exceptions.ServiceNotFoundException;
+import ai.wanaku.api.types.PromptReference;
+import ai.wanaku.api.types.io.PromptPayload;
+import ai.wanaku.core.mcp.common.resolvers.PromptsResolver;
+import ai.wanaku.core.persistence.api.PromptReferenceRepository;
+import jakarta.inject.Inject;
+import java.util.List;
+
+/**
+ * Wanaku implementation of PromptsResolver.
+ */
+public class WanakuPromptsResolver implements PromptsResolver {
+
+    @Inject
+    PromptReferenceRepository promptReferenceRepository;
+
+    @Override
+    public void provision(PromptPayload promptPayload) throws ServiceNotFoundException {
+        // For now, prompts don't require provisioning from external services
+        // This could be extended in the future if prompts need to fetch data from services
+    }
```

Comment:
I think we won't need this. Or, at least, not in the same way we have provisioning for add tools/resources. The reason being that, if capabilities were to provide prompts, then I believe they would provide it on connection. We can think a bit more about this when we start working on sampling and elicitation.